src/modules/storybook/stories/Section.stories.tsx

```diff
@@ -166,8 +166,7 @@ export const ListedSectionItems: Meta<SectionMetaProps> = {
                 <ToggleSectionItem
                   leftImage={<ThemeIcon svg={Warning} />}
                   text="ToggleSectionItem"
-                  isSubtextMarkdown={true}
-                  subtext={`1. This is a list \n 2. made with markdown`}
+                  subtext="Subtext"
```

Comment:
Markdown is not supported?

Reply:
Not anymore, no. We only used it on the the a11y screen (and here in storybook), and it had issues with screen reader. So I thought it was better to remove the feature.